File: .tmp_sc4013/java_06/2278_percentage_of_letter_in_string.java
// LeetCode 2278 - Percentage of Letter in String
// https://leetcode.com/problems/percentage-of-letter-in-string/

class Solution {
    public int percentageLetter(String s, char letter) {
        int cnt = 0;
        for (char c : s.toCharArray()) if (c == letter) cnt++;
        return cnt * 100 / s.length();
    }
}

File: .tmp_py4013/batch_02_sources/2278_percentage_of_letter_in_string.js
// LeetCode 2278 - Percentage of Letter in String
// https://leetcode.com/problems/percentage-of-letter-in-string/

var percentageLetter = function(s, letter) {
    let cnt = 0;
    for (const c of s) if (c === letter) cnt++;
    return Math.floor(cnt * 100 / s.length);
};
